documents-webapp/src/main/webapp/vue-app/documents/js/DocumentFileService.js

```diff
@@ -265,4 +265,17 @@ export function undoDeleteDocument(documentId) {
       throw new Error('Error when undoing deleting document');
     }
   });
+}
+
+export function getDocumentsTransferRules() {
```

Comment:
this method has to be moved inside https://github.com/exoplatform/ecms/blob/develop/apps/portlet-documents/src/main/webapp/js/transferRulesService.js and you will be able to use it by `this.$transferRulesService...` as it's defined in https://github.com/exoplatform/ecms/blob/develop/apps/portlet-documents/src/main/webapp/vue-app/attachment/initComponents.js#L59-L63

Reply:
done in this PR: https://github.com/exoplatform/ecms/pull/1787